src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/ECPoint.cs

```diff
@@ -9,7 +9,7 @@ namespace System.Security.Cryptography
     /// </summary>
     public struct ECPoint
     {
-        public byte[] X;
+        public byte[] X; // TODO not sure if it should be nullabe
```

Comment:
It's a mutable struct that people need to set the fields in.  That probably means nullable, since the reads aren't null-safe.